osu.Game.Tournament/Screens/SetupScreen.cs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Online.API;
+using osu.Game.Overlays;
+using osu.Game.Screens.Edit.Setup.Components.LabelledComponents;
+using osu.Game.Tournament.IPC;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Tournament.Screens
+{
+    public class SetupScreen : TournamentScreen, IProvideVideo
+    {
+        private FillFlowContainer fillFlow;
+
+        private LoginOverlay loginOverlay;
+
+        [Resolved]
+        private MatchIPCInfo ipc { get; set; }
+
+        [Resolved]
+        private IAPIProvider api { get; set; }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            InternalChild = fillFlow = new FillFlowContainer
+            {
+                RelativeSizeAxes = Axes.X,
+                AutoSizeAxes = Axes.Y,
+                Direction = FillDirection.Vertical,
+                Padding = new MarginPadding(10),
+                Spacing = new Vector2(10),
+            };
+
+            api.LocalUser.BindValueChanged(_ => Schedule(reload));
```

Comment:
I think it's fine triggering the change immediately instead of calling again
```suggestion
            api.LocalUser.BindValueChanged(_ => Schedule(reload), true);
```

Reply:
it changes behaviour. the initial load would be scheduled.